Contributor.md

```diff
@@ -1,3 +1,7 @@
+/Users/gaurav/PycharmProjects/hacktoberfest-1/Contributor.md
+- Name: Gaurav Pandey
+- Bio: I am an Engineering Student loves to code and develop.
+- Link: [Github](https://github.com/gauravpandey22)
```

Comment:
Gaurav  would you mind moving your lovely profile to bottom of the list? Thank you

Reply:
sure :)